packages/protocol/package.json

```diff
@@ -6,61 +6,61 @@
+    "revoke": "./scripts/bash/revoke.sh",
+    "set-block-gas-limit": "./scripts/bash/set_block_gas_limit.sh",
     "size:onchain": "./scripts/bash/get_smart_contract_size_from_onchain_address.sh",
     "size:artifacts": "./scripts/bash/get_smart_contract_size_from_build_artifacts.sh",
-    "ganache-dev": "./scripts/bash/ganache.sh",
-    "ganache-devchain": "./scripts/bash/ganache_devchain.sh",
+    "sourcify-publish": "yarn ts-node ./scripts/sourcify-publish.ts",
+    "submodules:pull": "git submodule update --init --recursive",
+    "submodules:delete": "rm -rf $(git submodule | awk '{ print $2 }')",
+    "tags:compare": "./scripts/bash/compare-git-tags.sh",
+    "tags:view": "git for-each-ref 'refs/tags/core-contracts.*' --sort=-committerdate --format='%(color:magenta)%(committerdate:short) %(color:blue)%(tree) %(color:green)github.com/celo-org/celo-monorepo/releases/tag/%(color:yellow)%(refname:short)'",
+    "test": "rm test/**/*.js ; node runTests.js",
+    "test:coverage": "yarn run test --coverage",
+    "test:gas": "yarn run test --gas",
+    "test:generate-old-devchain-and-build": "./scripts/bash/generate-old-devchain-and-build.sh",
+    "test:quicktest": "./scripts/bash/quicktest.sh",
+    "test:release-snapshots": "./scripts/bash/release-snapshots.sh",
+    "test:scripts": "yarn ts-node scripts/run-scripts-tests.ts --testPathPattern=scripts/",
     "truffle:migrate": "truffle migrate",
-    "devchain": "yarn ts-node scripts/devchain.ts",
-    "devchain:reset": "yarn devchain generate-tar .tmp/devchain.tar.gz --upto 29",
-    "anvil-devchain:start-L1": "./scripts/foundry/create_and_migrate_anvil_devchain.sh",
-    "anvil-devchain:start-L2": "./scripts/foundry/create_and_migrate_anvil_l2_devchain.sh",
-    "anvil-devchain:status": "if nc -z localhost 8546; then echo 'Devchain is serving at http://localhost:8546'; else echo 'Devchain is not running.'; fi",
-    "anvil-devchain:stop": "./scripts/foundry/stop_anvil.sh",
-    "anvil-devchain:e2e-tests": "./scripts/foundry/run_e2e_tests_in_anvil.sh",
-    "anvil-devchain:integration-tests": "./scripts/foundry/run_integration_tests_in_anvil.sh",
-    "view-tags": "git for-each-ref 'refs/tags/core-contracts.*' --sort=-committerdate --format='%(color:magenta)%(committerdate:short) %(color:blue)%(tree) %(color:green)github.com/celo-org/celo-monorepo/releases/tag/%(color:yellow)%(refname:short)'",
-    "generate-stabletoken-files": "yarn ts-node ./scripts/generate-stabletoken-files.ts",
-    "truffle-verify": "yarn truffle run verify",
-    "compare-git-tags": "./scripts/bash/compare-git-tags.sh"
+    "truffle:verify": "yarn truffle run verify",
+    "upgrade": "./scripts/bash/upgrade.sh",
+    "validate-abis-exports": "yarn ts-node --preferTsExts ./scripts/validate-abis-package.ts",
+    "verify:deployed": "./scripts/bash/verify-deployed.sh",
+    "verify:release": "./scripts/bash/verify-release.sh"
```

Comment:
Also connected with updating release documentation

Reply:
this will most probably be somehow rewritten once we update the release tooling for anvil

Reply:
Yes, but until we do, probably good to update the docs if we're cleaning up the package.json now?

Alternatively, keep all the legacy package.json scripts as they are, slated for removal soon with as the Foundry initiative progresses.

Reply:
Same as above ☝️

Reply:
Should be resolved 😄